docs/pages/guides/using-sentry.mdx

```diff
@@ -283,6 +169,39 @@ Once configured, information about the associated update will show up in an erro
 
 </Collapsible>
 
+## Sentry Integration with EAS
```

Comment:
Also, when I am trying to connect my Sentry account to Expo account, I got stuck on the below screen. I was doing this to test the connection features as described below. Do you think I am missing any steps?

![CleanShot 2025-04-22 at 17 08 28@2x](https://github.com/user-attachments/assets/1b81721a-9740-4ec2-98ec-6a8a2a189031)

Reply:
Thanks for sharing the context! I was lacking it.
The integration button is available to me in Expo dashboard.I have a staff account, which might be feature-flagged enabled on my end and that's why it might be showing under my Expo account. I'll re-check.